tool.go

```diff
@@ -266,6 +258,13 @@ func main() {
 		Use:   "run [gofiles...] [arguments...]",
 		Short: "compile and run Go program",
 	}
+	cmdRun.Flags().AddFlag(flagVerbose)
```

Comment:
This seems weird, creating this flag on `cmdBuild` and then adding it to `cmdRun`. I think we're approaching this wrong. What we probably should do instead is not use `Lookup` at all, but https://godoc.org/github.com/spf13/pflag#NewFlagSet, define flags on it and then https://godoc.org/github.com/spf13/pflag#FlagSet.AddFlagSet to add it to a command.